doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
-defined by [K8s Conformance](https://github.com/cncf/k8s-conformance) which is
-expected to grow according to the ongoing requirement traceability:
-- focus: \[Conformance\]
-- skip: \[Disruptive\]|NoExecuteTaintManager
-
-[Rally](https://github.com/openstack/rally) and is tool and framework that
-allows to perform Kubernetes API testing by iterating once the mainline
-[xrally-kubernetes](https://github.com/xrally/xrally-kubernetes) scenarios:
-
-[Functest xrally_kubernetes](http://artifacts.opnfv.org/functest-kubernetes/UFEMNKZPRBCO/functest-kubernetes-opnfv-functest-kubernetes-benchmarking-v1.21-xrally_kubernetes_full-run-16/xrally_kubernetes_full/xrally_kubernetes_full.html):
-
-| Scenarios                                                          |
-|--------------------------------------------------------------------|
-| Kubernetes.create_and_delete_deployment                            |
-| Kubernetes.create_and_delete_job                                   |
-| Kubernetes.create_and_delete_namespace                             |
-| Kubernetes.create_and_delete_pod                                   |
-| Kubernetes.create_and_delete_pod_with_configmap_volume             |
-| Kubernetes.create_and_delete_pod_with_configmap_volume [2]         |
-| Kubernetes.create_and_delete_pod_with_emptydir_volume              |
-| Kubernetes.create_and_delete_pod_with_emptydir_volume [2]          |
-| Kubernetes.create_and_delete_pod_with_hostpath_volume              |
-| Kubernetes.create_and_delete_pod_with_secret_volume                |
-| Kubernetes.create_and_delete_pod_with_secret_volume [2]            |
-| Kubernetes.create_and_delete_replicaset                            |
-| Kubernetes.create_and_delete_replication_controller                |
-| Kubernetes.create_and_delete_statefulset                           |
-| Kubernetes.create_check_and_delete_pod_with_cluster_ip_service     |
-| Kubernetes.create_check_and_delete_pod_with_cluster_ip_service [2] |
-| Kubernetes.create_check_and_delete_pod_with_node_port_service      |
-| Kubernetes.create_rollout_and_delete_deployment                    |
-| Kubernetes.create_scale_and_delete_replicaset                      |
-| Kubernetes.create_scale_and_delete_replication_controller          |
-| Kubernetes.create_scale_and_delete_statefulset                     |
-| Kubernetes.list_namespaces                                         |
+Then CNTT RC2 starts with the test list defined by
+[K8s Conformance](https://github.com/cncf/k8s-conformance) which is expected to
+grow according to the ongoing requirement traceability.
 
-The following software versions are considered to verify Kubernetes v1.21
-(latest stable release) selected by CNTT:
+[End-to-End Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md)
+basically respectively asks focus and fix regexes to select or to blacklist
```

Comment:
not sure what is intended here -- suggest delete "respectively"
what is meant by "asks focus and fix regexes"?